python/ray/serve/tests/test_autoscaling_policy.py

```diff
@@ -1205,7 +1205,9 @@ def g():
     }
 
     client.deploy_apps(ServeDeploySchema(**{"applications": [app_config]}))
-    wait_for_condition(lambda: client.get_serve_status().app_status.status == "RUNNING")
+    wait_for_condition(
+        lambda: serve.status().applications["default"].status == "RUNNING"
```

Comment:
Non-blocker: Kinda nitpick to maybe switch all the usage of `"default"` to `SERVE_DEFAULT_APP_NAME` XP

Reply:
Good point, I've replaced the raw string with the constant!